feature_engine/features/temporal_features.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+
+def mean_conv_msg_duration(df,on_column):
+
+
+    #convert timestamp column to datetime type (in minutes)
+    df[on_column] = pd.to_datetime(df[on_column])
+
+    # calculate difference between consecutive timestamps
+    df['time_diff'] = df[on_column].diff().dt.total_seconds()/60
+    return  df['time_diff'].mean() 
```

Comment:
This currently returns the mean and SD of the entire column --- which means it ends up being constant *across* conversations.